scripts/globals/spells/blue/amplification.lua

```diff
@@ -28,33 +28,18 @@ spellObject.onSpellCast = function(caster, target, spell)
     local typeEffectOne = xi.effect.MAGIC_ATK_BOOST
     local typeEffectTwo = xi.effect.MAGIC_DEF_BOOST
     local power = 10
-    local duration = 90
+    local duration = bluGetDurationWithDiffusion(caster, 90)
     local returnEffect = typeEffectOne
 
-    if caster:hasStatusEffect(xi.effect.DIFFUSION) then
-        local diffMerit = caster:getMerit(xi.merit.DIFFUSION)
+    local actionOne = target:addStatusEffect(typeEffectOne, power, 0, duration)
+    local actionTwo = target:addStatusEffect(typeEffectTwo, power, 0, duration)
 
-        if diffMerit > 0 then
-            duration = duration + (duration / 100) * diffMerit
-        end
-
-        caster:delStatusEffect(xi.effect.DIFFUSION)
-    end
-
-    if
-        not target:addStatusEffect(typeEffectOne, power, 0, duration) and
-        not target:addStatusEffect(typeEffectTwo, power, 0, duration)
-    then
-        -- both statuses fail to apply
+    if not actionOne and not actionTwo then -- both statuses fail to apply
         spell:setMsg(xi.msg.basic.MAGIC_NO_EFFECT)
-    elseif not target:addStatusEffect(typeEffectOne, power, 0, duration) then -- the first status fails to apply
-        target:addStatusEffect(typeEffectTwo, power, 0, duration)
-        spell:setMsg(xi.msg.basic.MAGIC_GAIN_EFFECT)
+    elseif not actionOne and actionTwo then -- the first status fails to apply 
         returnEffect = typeEffectTwo
-    else
-        target:addStatusEffect(typeEffectOne, power, 0, duration)
-        target:addStatusEffect(typeEffectTwo, power, 0, duration)
-        spell:setMsg(xi.msg.basic.MAGIC_GAIN_EFFECT)
+    elseif actionOne and not actionTwo then -- the second status fails to apply 
+        returnEffect = typeEffectOne
```

Comment:
Just yesterday I found a message basic that has space to return 2 status effects. This may be a good place to use it.

This is more a note to myself than anything else